docs/versions/latest/Release.md

```diff
@@ -81,6 +81,20 @@ If you build project with polyglot + typescript-project-all the build will succe
 
 None
 
+### Fix error Command line is too long
```

Comment:
The previous and new behaviour are pretty obvious, can you explain more about **how** it was done?

Reply:
Thank you for your review 
Will update **New Behavior** with below comment:
**If you build project with any number of dependencies the build will succeed since we are splitting the command so that each command can have max 7000 characters**

Please let me know if we need to add more details

Reply:
Addressed